src/pages/admin/AdminCustomerInstitutionPage.tsx

```diff
@@ -39,6 +39,12 @@ const AdminCustomerInstitutionPage: FC = () => {
     editMode
   );
 
+  useEffect(() => {
+    if (customerInstitution) {
+      history.replace(`/admin-institutions/${customerInstitution.identifier}`, { title: customerInstitution.name });
```

Comment:
aha, dette er state.title!
ville antatt at history.replace endrer URLen, men den setter kanskje bare ting i state?

Reply:
Det endrer historien, og legger på state, så når du går "tilbake", så kommer du til den siden før du kom til admin-customer/12321, isteden for at når du går tilbake så kommer du til admin-customer/12321.

Bedre forklart her: https://stackoverflow.com/questions/39340108/what-is-the-trade-off-between-history-push-and-replace 😅